plugins/hydra_ray_launcher/examples/multi_module/train.py

```diff
@@ -8,7 +8,7 @@
 log = logging.getLogger(__name__)
 
 
-@hydra.main(config_path="conf", config_name="config")
```

Comment:
Please place your configs in a conf directory.
There are some nasty side effects to not having a config dir: anything under your Python file becomes eligible to be imported as a config. any directory with yaml files becomes a config group etc. this is not good practice.